src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package by.itechart.api.controller;
+
+import by.itechart.api.entity.User;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/user")
```

Comment:
Why did you decide to use a singular form of  `user`?

Reply:
I think that every operation here will be related to a user, that's why I decided to do that.